source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidRealTimeMultiplayerClient.cs

```diff
@@ -34,111 +54,719 @@ public AndroidRealTimeMultiplayerClient(AndroidJavaObject account)
+
+                            using (var roomStatusUpdateCallback =
+                                new AndroidJavaObject("com.google.games.bridge.RoomStatusUpdateCallbackProxy",
+                                    new RoomStatusUpdateCallbackProxy(this, listener)))
+                            using (roomConfigBuilder.Call<AndroidJavaObject>("setRoomStatusUpdateCallback",
+                                roomStatusUpdateCallback))
+                                ;
+
+                            using (roomConfigBuilder.Call<AndroidJavaObject>("addPlayersToInvite",
+                                AndroidJavaConverter.ToJavaStringList(result.PlayerIdsToInvite)))
+                            {
+                                mRoomConfig = roomConfigBuilder.Call<AndroidJavaObject>("build");
+                            }
+
+                            mListener = listener;
+                        }
+
+                        using (var task = mClient.Call<AndroidJavaObject>("create", mRoomConfig))
+                        {
+                            AndroidTaskUtils.AddOnFailureListener(
+                                task,
+                                exception =>
+                                {
+                                    listener.OnRoomConnected(false);
+                                    CleanSession();
+                                });
+                        }
+                    }
+                });
+        }
+
+        private int GetMinParticipantsToStart()
+        {
+            int minParticipantsToStart;
+            using (var participants = mRoom.Call<AndroidJavaObject>("getParticipants"))
+            {
+                minParticipantsToStart = participants.Call<int>("size");
+            }
+
+            using (var autoMatchCriteria = mRoom.Call<AndroidJavaObject>("getAutoMatchCriteria"))
+            {
+                if (autoMatchCriteria != null)
+                {
+                    minParticipantsToStart = minParticipantsToStart +
+                                             autoMatchCriteria.Call<int>("getInt", "min_automatch_players", 0);
                 }
             }
 
-            using (var task = mClient.Call<AndroidJavaObject>("create", mRoomConfig))
+            if (mInvitation != null)
             {
+                minParticipantsToStart = minParticipantsToStart + 1;
             }
+
+            return minParticipantsToStart;
         }
 
-        public void CreateWithInvitationScreen(uint minOpponents, uint maxOppponents, uint variant,
-            RealTimeMultiplayerListener listener)
+        private float GetPercentComplete()
         {
-            // Task<Intent> getSelectOpponentsIntent(@IntRange(from = 1) int minPlayers, @IntRange(from = 1) int maxPlayers, boolean allowAutomatch)
+            int connectedCount = GetConnectedParticipants().Count;
+            float percentPerParticipant = (100.0F - InitialPercentComplete) / GetMinParticipantsToStart();
```

Comment:
Probably it can be more than 100 in this case if GetConnectedParticipants is bigger than GetMinParticipantsToStart

Reply:
Updated as
return Math.Min(100F * connectedPlayerCount / mMinPlayersToStart, 100F);